__tests__/app.test.js

```diff
@@ -323,3 +323,25 @@ describe("PATCH /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("DELETE:204 returns status 204 and has deleted corresponding comment", () => {
+    return request(app).delete("/api/comments/3").expect(204);
```

Comment:
would be good to check the response here too as we would expect it to be no content/an empty object